.gitignore

```diff
@@ -6,6 +6,7 @@
 *$
 *.bak
 *.diff
+.idea
```

Comment:
Thanks for checking. Not in this case though. It's the folder name just as `.git`.